jsonrpc/types_test.go

```diff
@@ -90,7 +90,7 @@ func TestDecode_TxnArgs(t *testing.T) {
 		assert.NoError(t, json.Unmarshal(buffer.Bytes(), &r))
 
 		if !reflect.DeepEqual(r, c.res) {
-			t.Fatal("bad")
+			t.Fatal("Resulting data and expected values are not equal")
```

Comment:
Thank you for renaming this error. I've searched our code, there are multiple places with this error message which we should refactor ("bad")

Reply:
I think this is the 3rd time I've been saying this on multiple PRs, this sort of refactor is out of the scope for this PR and should be tackled in the future - I simply modified a test chunk of code I actually tested out and changed

Reply:
agreed, we will tackle this some other time